spec/lib/appsignal/integrations/mongo_ruby_driver_spec.rb

```diff
@@ -71,6 +72,16 @@
         store[2] = command
       end
 
+      it "should emit a measurement" do
+        expect(Appsignal).to receive(:add_distribution_value).with(
+          "mongodb_query_duration",
+          0.9919,
+          :database => "test"
+        )
```

Comment:
best would be to call `.and_call_original` afterwards so we know when it errors on the actual call. This stub doesn't catch mismatched arguments and such.